### ISSUE TITLE
Running inference over .tif in tf_od_predict.py

### ISSUE THREAD

**Issue by Geo-Trackers** [developmentseed/label-maker#127]
Can I run inference over the image tiles which is in .tif format?

Thank you. I look forward to hear from you.

Best, 

Suman

**Comment by Geoyi** [comment]
@Geo-Trackers, yes, you can run the inference over the image tile, and the [main script is this line](https://github.com/developmentseed/label-maker/blob/7b8953c6f7d6dce24fa3eed4a1b6179b60b7db4f/examples/utils/tf_od_predict.py#L93). You can replace `.jpg` with `.tif`.

**Comment by Geoyi** [comment]
Yes, @Geo-Trackers. But it's not very straightforward. 

When you use Label Maker to generate `.tif` tile, each tile come with an ID, e.g. `207692-115930-18.tif`, you can use [Mercantile](https://github.com/mapbox/mercantile) e.g., `mercantile.bounds` to get the tile geo-coordinates, and each bbox within the tile can be computed through their associated tile ID if that makes sense.
